Core/HLE/sceDisplay.cpp

```diff
@@ -57,6 +57,12 @@
 #include "GPU/Common/PostShader.h"
 #include "GPU/Debugger/Record.h"
 
+#ifdef OPENXR
+#define FRAMELIMIT 72
+#else
+#define FRAMELIMIT 60
+#endif
+
```

Comment:
This changes the internal framerate of the PSP, which will work in some games to some degree but can cause timing errors in some. Is it possible to switch the headset to 60Hz mode? I realize that displaying 60hz in 72hz might look a bit choppy, but if we do, doesn't the XR driver actually compensate by rendering prediction-distorted frames so that head movement is at least smooth?

Reply:
I invested here a lot of time. Running the headset on 60Hz is painful for the eyes (at least on Quest). With 60Hz I got almost immediately headaches, with 72Hz I am okay with session over one hour. My testers checked about 70 games (the list of test results is in a pinned message on #ppsspp-vr-testing). None of them complained about timing related issues.

With 60Hz PPSSPP and 72Hz OpenXR prediction distorted frames is there a problem that the view matrix for the scene is not updated and makes the user experience even worse.

How strongly are you against this change? I would really love to keep it.

Reply:
Games that don't try to check real time will run 20% too fast like this, that's not an issue?

Also, not all VR headsets are 72hz, so this is very Oculus-specific. 

I do fear this will cause all sorts of weird compatibility issues, even if games seem fine with it, they might crash or misbehave at some point later in game, which might lead to spurious bug reports.

But if there really is no other way...

Reply:
I understand the problem. There might exist a better solution but I invested a lot of time into it and didn't find a better solution.

I will be happy to look into it again once anyone find a game where it causes issues.